src/pages/strategies/create/recurring.tsx

```diff
@@ -0,0 +1,122 @@
+import { useSearch } from '@tanstack/react-router';
+import { useTokens } from 'hooks/useTokens';
+import { StrategySettings } from 'libs/routing';
+import { CreateOrder } from 'components/strategies/create/CreateOrder';
+import { OrderBlock } from 'components/strategies/common/types';
+import { useMarketPrice } from 'hooks/useMarketPrice';
+import { CreateLayout } from 'components/strategies/create/CreateLayout';
+import { CreateForm } from 'components/strategies/create/CreateForm';
+import { useSetRecurringOrder } from 'components/strategies/common/useSetOrder';
+import {
+  isZero,
+  outSideMarketWarning,
+} from 'components/strategies/common/utils';
+
+export interface CreateRecurringStrategySearch {
+  base: string;
+  quote: string;
+  buyMin?: string;
+  buyMax?: string;
+  buyBudget?: string;
+  buySettings: StrategySettings;
+  sellMin?: string;
+  sellMax?: string;
+  sellBudget?: string;
+  sellSettings: StrategySettings;
+}
+
+type Search = Omit<CreateRecurringStrategySearch, 'base' | 'quote'>;
+
+const getWarning = (search: CreateRecurringStrategySearch) => {
+  const { buyMin, buyMax, sellMin, sellMax } = search;
+  const sellMinInRange =
+    buyMin && buyMax && sellMin && +sellMin >= +buyMin && +sellMin < +buyMax;
+  const buyMaxInRange =
+    sellMin && sellMax && buyMax && +buyMax >= +sellMin && +buyMax < +sellMax;
+  if (sellMinInRange || buyMaxInRange) {
+    return 'Notice: your Buy and Sell orders overlap';
+  }
+};
+
+const getError = (search: CreateRecurringStrategySearch) => {
+  const { buyMin, buyMax, sellMin, sellMax } = search;
+  const minReversed = buyMin && sellMin && +buyMin > +sellMin;
+  const maxReversed = buyMax && sellMax && +buyMax > +sellMax;
+  if (minReversed || maxReversed) {
+    return 'Orders are reversed. This strategy is currently set to Buy High and Sell Low. Please adjust your prices to avoid an immediate loss of funds upon creation.';
+  }
+};
```

Comment:
Can you use checkIfOrdersReversed from src/components/strategies/utils.ts here instead? To keep the check consistent throughout the app